adapters/Pangle/PangleAdapter/GADMediationAdapterPangleConstants.h

```diff
@@ -14,11 +14,15 @@
 
 #import <Foundation/Foundation.h>
 
-/// Pangle mediation network adapter version.
-static NSString *const GADMAdapterPangleVersion = @"x.y.z.p";
+/// Pangle adapter version.
+static NSString *const GADMAdapterPangleVersion = @"1.5.0.0";
```

Comment:
Since the latest version of `Ads-Global` is `4.3.0.2`:
https://github.com/bytedance/Bytedance-UnionAD/blob/master/Ads-Global/Ads-Global.podspec

Please update this to `4.3.0.2.0`.

Reply:
this is Pangle adapter version.

Reply:
"GADMAdapterPangleVersion" is  Pangle adpater Version. Pangle SDK versions have 4 digits.

Reply:
Since Pangle SDK versions have four digits, we need to have five digits in adapter version.